src/main/java/com/neprozorro/exceptions/InvalidInputFormatException.java

```diff
@@ -0,0 +1,14 @@
+package com.neprozorro.exceptions;
+
+import java.math.BigDecimal;
+
+public class InvalidInputFormatException extends RuntimeException{
+
+    public InvalidInputFormatException(){
+        super("Invalid input format. You have to paste two values. Paste min and max total price");
+    }
+
+    public InvalidInputFormatException(BigDecimal minValue, BigDecimal maxValue){
```

Comment:
I`m not sure, but it will be 2 different exceptions, instead of one with two constructors.

Reply:
Why is it InvalidInputFormatException and not IllegalArgumentException? The last one looks more popular but the meaning of both are clear. Why did you choose the InvalidInputFormatException? You should have a reason for everything you do and be able to provide arguments. 
And also it should be 2 parameters where first parameter is a message and the second is a value. Like ("Value {} is not valid", value) or it's something like this, you can check in the project how it's done. Use ctrl+shift+F to look code pieces in all project, you should use it really often.

Reply:
> I`m not sure, but it will be 2 different exceptions, instead of one with two constructors.

No, there are a couple of constructors in Exceptions, we can provide more than one constructor to our exceptions as well.